simpl/config.py

```diff
@@ -165,10 +165,13 @@ def main(argv):
 import argparse
 import collections
 import copy
+import errno
 import logging
 import os
 import sys
 
+from simpl import exceptions as simpl_exceptions
```

Comment:
It would be nice if we came up with a convention for importing exceptions.

Some things we've discussed before that maybe we can add to a contributing.md or in the docstring for simpl/exceptions.py:
- A fully-named exception is useful when exceptions are caught and bubble up (ex. SimplConfigError is easier to track than simpl.ConfigError if it bubbles out of simpl)
- "exceptions" is a long word. One of our projects puts exceptions in `exc.py`; I think that's a good idea.
- valid reasons for renaming imports are to avoid conflicts or to shorten very long names.
simple_exceptions is no shorter than simpl.exceptions. So why not just:
```python
import simple.exceptions

...
raise simpl.exceptions.SimplConfigUnusedIniOption()
```